bindings/python/tools/python/ostk/astrodynamics/dataframe.py

```
@@ -595,3 +598,92 @@ def _get_entry_from_state(
+            - 'timestamp': UTC datetime of the state (Python datetime object)
+            - 'dr': Position residual in meters (L2 norm of position difference)
+            - 'dr_x': Position residual x-component in meters
+            - 'dr_y': Position residual y-component in meters
+            - 'dr_z': Position residual z-component in meters
+            - 'dv': Velocity residual in meters/second (L2 norm of velocity difference)
+            - 'dv_x': Velocity residual x-component in meters/second
+            - 'dv_y': Velocity residual y-component in meters/second
+            - 'dv_z': Velocity residual z-component in meters/second
+    """
+    instants: list[Instant] = [s.get_instant() for s in states]
+    datetimes: list[datetime] = [i.get_date_time(Scale.UTC) for i in instants]
```

Ruff (0.14.3)

Local variable `datetimes` is assigned to but never used

Remove assignment to unused variable `datetimes`

(F841)